DQMOffline/Alignment/interface/TkAlCaRecoMonitor.h

```diff
@@ -36,20 +36,33 @@ class TkAlCaRecoMonitor : public DQMEDAnalyzer {
   explicit TkAlCaRecoMonitor(const edm::ParameterSet &);
   ~TkAlCaRecoMonitor() override = default;
 
+  static void fillDescriptions(edm::ConfigurationDescriptions &);
+
   void bookHistograms(DQMStore::IBooker &, edm::Run const &, edm::EventSetup const &) override;
   void analyze(const edm::Event &, const edm::EventSetup &) override;
 
 private:
+  static constexpr const double kMuonMass_ = 0.10565836;
```

Comment:
Ultra minor point: it is funny, there are slightly different values of the muon mass scattered in our code. The PDG value is 0.1056583755, tinily larger than what written here, Lacking a centrally accessible value of it in CMSSW, one could stick on the PDG value

Reply:
I would be really glad if we actually had to care about the sub per million differences in the muon mass discrepancy in the di-muon resonance mass that we are monitoring. We have much worse problems than this :D

Reply:
In any case done at https://github.com/cms-sw/cmssw/pull/40887.